src/enrichers/idcookie.ts

```diff
@@ -1,23 +1,19 @@
 import { WrappedReadOnlyStorageHandler } from '../handlers/storage-handler'
-import { Enricher, IdCookieConfig } from '../types'
+import { Enricher, State } from '../types'
 
-type Input = { idCookie?: IdCookieConfig, peopleVerifiedId?: string }
-type Output = { resolvedIdCookie: string | null }
+type Input = Pick<State, 'idCookie' | 'peopleVerifiedId'>
```

Comment:
This is a bit nicer than typing things out manually. Will migrate the other modules later